### DISCUSSION TITLE
Zigbee

### DISCUSSION THREAD

**Discussion by alessio1979** [indomus/forum#494] [ZigBee, Z-Wave, Thread, Matter]
Buongiorno ho un problema con uno switch dimmer meros zigbee ovvero il coordinatore c2531 lo accoppia ma dice che il device non è supportato, grazie.

**Comment by snakuzzo** [comment]
CIao

1. Modifica il titolo del topic altrimenti non si capisce un tubo
2. Se non è supportato segui la procedura per farlo integrare su zigbee2mqtt

https://www.zigbee2mqtt.io/how_tos/how_to_support_new_devices.html

**Reply by alessio1979** [reply]
Ho provato a seguire quella procedura ma non mi è tanto chiaro il da farsi

**Reply by snakuzzo** [reply]
Devi semplicemente seguire passo passo quel che c'è scritto

**Reply by alessio1979** [reply]
Ok stasera ci provo grazie mille

**Reply by alessio1979** [reply]
Ci riprovo

**Reply by KC-inDomus** [reply]
Alessio si legge il tuo fqdn, cancella l'immagine.
